charts/nx-cloud/templates/nx-cloud-api-service.yaml

```diff
@@ -5,6 +5,7 @@ metadata:
   name: nx-cloud-nrwl-api-service
   labels:
     app: nx-cloud-api
+    testing: pipeline-testing
```

Comment:
This is only to trigger the pipeline during testing, it will be removed prior to merge